document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/LogicOperator.java

```diff
@@ -1,6 +1,6 @@
 package org.hypertrace.core.documentstore.expression.operators;
 
-public enum LogicalOperator {
+public enum LogicOperator {
```

Comment:
I think `LogicalOperator` is more easily understandable.
Refer terminology from https://www.geeksforgeeks.org/operators-in-java/.